test/python/test_quantumprogram.py

```diff
@@ -1532,6 +1532,31 @@ def test_offline(self):
         # SDK will throw ConnectionError on every call that implies a connection
         self.assertRaises(ConnectionError, qp.set_api, FAKE_TOKEN, FAKE_URL)
 
+    def test_reconfig(self):
+        """Test reconfiguring the qobj from 1024 shots to 2048 using
+        reconfig instead of recompile
+        """
+        QP_program = QuantumProgram(specs=self.QPS_SPECS)
+        qr = QP_program.get_quantum_register("qname")
+        cr = QP_program.get_classical_register("cname")
+        qc2 = QP_program.create_circuit("qc2", [qr], [cr])
+        qc2.measure(qr[0], cr[0])
+        qc2.measure(qr[1], cr[1])
+        qc2.measure(qr[2], cr[2])
+        circuits = ['qc2']
```

Comment:
We don't really need this variable, right?

Reply:
Fixed